frontend/app/src/components/dashboard/NftBalanceTable.vue

```diff
@@ -182,18 +198,59 @@ const percentageOfTotalNetValue = (value: BigNumber) => {
 };
 
 const percentageOfCurrentGroup = (value: BigNumber) => {
-  return calculatePercentage(value, get(total));
+  return calculatePercentage(value, get(totalUsdValue) as BigNumber);
 };
 
 const refresh = async () => {
-  return await fetchNonFungibleBalances({ ignoreCache: true });
+  return await fetchNonFungibleBalances(true);
 };
 
 const { dashboardTablesVisibleColumns } = storeToRefs(
   useFrontendSettingsStore()
 );
 
-const filteredBalances = computed(() => {
-  return get(nonFungibleBalances).filter(item => !item.isLp);
+const { data, limit, found, total, totalUsdValue } =
```

Comment:
Any idea how much of this pagination code is duplicated across the table components? do you think we could create a generic composable to reuse at least the common part of the functionality?